lib/davinci_dtr_test_kit/dtr_light_ehr_suite.rb

```diff
@@ -125,16 +136,106 @@ class DTRLightEHRSuite < Inferno::TestSuite
         oauth_credentials :credentials
       end
 
-      group from: :questionnaire_response_group
-      group from: :coverage_group
-      group from: :communication_request_group
-      group from: :device_request_group
-      group from: :encounter_group
-      group from: :medication_request_group
-      group from: :nutrition_order_group
-      group from: :service_request_group
-      group from: :task_group
-      group from: :vision_prescription_group
+      group from: :questionnaire_response_group,
+            config: {
+              inputs: {
+                id_token: { name: :ehr_id_token },
+                client_id: { name: :ehr_client_id },
+                requested_scopes: { name: :ehr_requested_scopes },
+                access_token: { name: :ehr_access_token },
```

Comment:
This works, but it means that
1. only the token from the ehr launch can get used
2. you had to change all of these tests

Instead, I think we should take the approach that (g)(10) takes to this and rename the output so that both use "access_token" and whichever one got run most recently will be used. See https://github.com/onc-healthit/onc-certification-g10-test-kit/blob/main/lib/onc_certification_g10_test_kit/smart_standalone_patient_app_group.rb#L534 for an example

Reply:
Addressed in [Change name of output rather than input](https://github.com/inferno-framework/davinci-dtr-test-kit/pull/45/commits/716da6064d168a9174e48baba2b8f8f5c91f65b4)